src/js/components/widget.jsx

```diff
@@ -33,6 +33,10 @@ export class WidgetComponent extends Component {
         ui: PropTypes.object.isRequired
     };
 
+    state = {
+        showAnimation: false
```

Comment:
mainly because you need to handle the case where someone calls `Smooch.open()` and `Smooch.close()`